### ISSUE TITLE
Iphone crash

### ISSUE THREAD

**Issue by pfh59** [pfh59/eve-whmapper#107]
**Describe the bug**
After logging on iPhone, an error appear

**To Reproduce**
1. ha ve an iPhone
2. Click on Account/Login
3. Type your eve user password
4. Mapper loading
5. See error

**Expected behavior**
No error

**Smartphone (please complete the following information):**
 - Device: iPhone XS
 - OS: Ios 16.5.1
 - Browser : Safari

**Comment by pfh59** [comment]
**Logs :** 

```
Unhandled exception rendering component: Unable to cast object of type 'Microsoft.AspNetCore.Components.Web.TouchEventArgs' to type 'Microsoft.AspNetCore.Components.Web.MouseEventArgs'.
whmapper-beta_1  |       System.InvalidCastException: Unable to cast object of type 'Microsoft.AspNetCore.Components.Web.TouchEventArgs' to type 'Microsoft.AspNetCore.Components.Web.MouseEventArgs'.
whmapper-beta_1  |          at MudBlazor.MudMenu.OpenMenu(EventArgs args)
whmapper-beta_1  |          at MudBlazor.MudMenu.ToggleMenuTouch(TouchEventArgs args)
whmapper-beta_1  |          at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
whmapper-beta_1  |          at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
whmapper-beta_1  |       --- End of stack trace from previous location ---
whmapper-beta_1  |          at Microsoft.AspNetCore.Components.ComponentBase.CallStateHasChangedOnAsyncCompletion(Task task)
whmapper-beta_1  |          at Microsoft.AspNetCore.Components.RenderTree.Renderer.GetErrorHandledTask(Task taskToHandle, ComponentState owningComponentState)
whmapper-beta_1  | fail: Microsoft.AspNetCore.Components.Server.Circuits.CircuitHost[111]
whmapper-beta_1  |       Unhandled exception in circuit 'z2Vw_rW00S4pB2d8Y3-lFd75qFiKl52pCN9YM6OLWfg'.
whmapper-beta_1  |       System.InvalidCastException: Unable to cast object of type 'Microsoft.AspNetCore.Components.Web.TouchEventArgs' to type 'Microsoft.AspNetCore.Components.Web.MouseEventArgs'.
whmapper-beta_1  |          at MudBlazor.MudMenu.OpenMenu(EventArgs args)
whmapper-beta_1  |          at MudBlazor.MudMenu.ToggleMenuTouch(TouchEventArgs args)
whmapper-beta_1  |          at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
whmapper-beta_1  |          at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
whmapper-beta_1  |       --- End of stack trace from previous location ---
whmapper-beta_1  |          at Microsoft.AspNetCore.Components.ComponentBase.CallStateHasChangedOnAsyncCompletion(Task task)
whmapper-beta_1  |          at Microsoft.AspNetCore.Components.RenderTree.Renderer.GetErrorHandledTask(Task taskToHandle, ComponentState owningComponentState)
```

**Comment by pfh59** [comment]
bug open : https://github.com/MudBlazor/MudBlazor/issues/8125

**Comment by pfh59** [comment]
bug seems to be fixed since v6.19.1
